File: Python Basics/while_loop/account_balance.py
entry = input()
total_sum = 0
while entry != 'NoMoreMoney':
    entry = float(entry)
    if entry < 0:
        print('Invalid operation!')
        break
    total_sum += entry
    print(f'Increase: {entry:.2f}')
    entry = input()

print(f'Total: {total_sum:.2f}')

File: JS/While Loop/accountBalance.js
function accountBalance(input){
    let idx = 0;
    let income = input[idx];
    let balance = 0
    while(income != 'NoMoreMoney'){
        if(Number(income) < 0){
            console.log('Invalid operation!')
            break;
        }
        balance = balance + Number(income);
        console.log(`Increase: ${Number(income).toFixed(2)}`);
        idx++;
        income = input[idx];
    }
    console.log(`Total: ${balance.toFixed(2)}`)
}

// accountBalance(["120",
// "45.55",
// "-150"])

